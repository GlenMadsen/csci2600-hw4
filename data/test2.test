a
r
7
11
r
81
20
r
Lally
Lally Hall
r
Troy Building
11
r
Troy Building
87 Gymnasium
q