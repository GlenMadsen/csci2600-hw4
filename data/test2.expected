Unknown option
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to 87 Gymnasium:
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
Total distance: 156.051 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from RPI Ambulance to Cogswell Laboratory:
	Walk North to (Intersection 99)
	Walk NorthWest to (Cogswell Laboratory)
Total distance: 143.488 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Lally]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to 87 Gymnasium:
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
Total distance: 156.051 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to 87 Gymnasium:
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
Total distance: 156.051 pixel units.
